scripts/bluepill.sh

```diff
@@ -126,21 +147,6 @@ bluepill_runner_tests()
   fi
 }
 
-bluepill_integration_tests()
```

Comment:
So are we not doing integration tests anymore or did these get moved somewhere else?